components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIManager.java

```diff
@@ -439,11 +439,11 @@ String addResourceFile(Identifier identifier, String resourcePath, ResourceFile
      * Check whether the given scope key is already available under given tenant
      *
      * @param scopeKey candidate scope key
-     * @param tenantid tenant id
+     * @param tenantDomain tenant domain
      * @return true if the scope key is already available
      * @throws APIManagementException if failed to check the context availability
      */
-    boolean isScopeKeyExist(String scopeKey, int tenantid) throws APIManagementException;
+    boolean isScopeKeyExist(String scopeKey, String tenantDomain) throws APIManagementException;
```

Comment:
Can we really change these APIs directly? I think it is better to introduce another method.

Reply:
Agree. Fixed with https://github.com/wso2/carbon-apimgt/pull/8409/commits/ada153b9d08f3bc347497fc0e38e11d7ef04a8b2 and https://github.com/wso2/carbon-apimgt/pull/8409/commits/72c63ac9a76e5af0dda048e97f742c9d361b4259